src/dropdowns/dropdown-trigger.ts

```diff
@@ -0,0 +1,46 @@
+import {Directive, HostListener, ElementRef} from 'angular2/core';
+import {NglDropdown} from './dropdown';
+
+@Directive({
+  selector: '[nglDropdownTrigger]',
+  host: {
+    'aria-haspopup': 'true',
+    'tabindex': '0',
```

Comment:
Indeed this happens because `<a>`s don't have an `href`. For Angular application an `href` may cause problems, so we cannot ask from people to add it. Let's keep the `tabindex` helper :+1: